content/docs/0.11.x/reference/bridge/oauth/index.md

```diff
@@ -199,37 +286,40 @@ definitions:
 
 The Keptn Bridge expects the environment variable `OAUTH_DISCOVERY`. This must direct to a discovery endpoint with the following details:
 
-1. Endpoint for authorization request generation - `authorization`
-1. Endpoint to handle tokens and provide the login decision - `token_decision`
+1. Endpoint for authorization request generation - `authorization_endpopint`.
+2. Endpoint to handle tokens and provide the login decision - `token_endpoint`.
+3. Endpoint to fetch JSON Web key set to validate the ID token - `jwks_uri`.
+4. (optional) Endpoint to handle the logout - `end_session_endpoint`.
 
 With above-mentioned keys, the following is a sample response of the discovery
 
 ```
-Get http://oauth-service:8080/discovery
+Get http://identity-provider:8080/.well-known/openid-configuration
 
 {
- "authorization": "http://oauth-service:8080/authorization",
- "token_decision": "http://oauth-service:8080/token_decision"
+ "authorization_endpoint": "http://identity-provider:8080/authorization",
+ "token_endpoint": "http://identity-provider:8080/token_decision"
+ "jwks_uri": "http://identity-provider:8080/jwks"
 }
 ```
 
 From the response, Keptn bridge identifies the specific endpoints it needs to consume. Following sections provide specific details of these endpoints.
 
 ### authorization endpoint
+Generates an authentication code and redirects to the given redirect URI.
 
-Generates an authorization request with correct values and return that with the key *authorization_url* 
+The following query parameters must be supported:
+- client_id
+- redirect_uri
+- scope
+- state
+- nonce
+- code_challenge
+- code_challenge_method
 
-```
-Get http://oauth-service:8080/authorization
+### token endpoint
 
-{
- "authorization_url": "http://idp.com/authorization?client_id=xyz&redirect_uri=http://keptn.com/oauth/redirect&scope=openid&state=123"
-}
-```
-
-### token_decision endpoint
-
-Consumes code and token of the authorization response and provides the login decision. For simplicity, following are the currently supported decisions:
+Consumes code and state of the authorization response and provides the login decision. For simplicity, following are the currently supported decisions:
```

Comment:
```suggestion
It consumes the code challenge and state of the authorization response and provides the login decision. For simplicity, the following are the currently supported decisions:
```

Reply:
It's the `authorization code`, not the `code challenge`. The `code challenge` is sent to the authorization endpoint, not to the token endpoint.

Reply:
Thank you for the clarification! :)